yang/frr-bgp.yang

```diff
@@ -405,7 +409,7 @@ module frr-bgp {
       description
         "A list of network routes.";
       leaf prefix {
-        type rt-types:ipv4-multicast-group-address;
+        type frr-pim-rp:ipv4-multicast-group-address-prefix;
```

Comment:
It looks like `aggregate-route` and `admin-distance-route`.
And the same three leafs for IPv6.